### ISSUE TITLE
Select a11y issues

### ISSUE THREAD

**Issue by PahaN47** [gravity-ui/uikit#1760]
The current implementation of the `Select` component does implement a11y features related to reading the currently active option out loud using the `aria-activedescendant` attribute. However, there still exist several issues at the moment:
- This feature does not seem to be working in Safari when using VoiceOver (might not be fixable)
- It is impossible to get active descendant's id without looking directly at the source code when using custom controls as the current option's id is not passed to the `renderControl` function
- When using a filterable `Select` where the filter automatically receives focus through a ref, `aria-activedescendant` attribute does not take effect since the control that has it loses focus. I suggest that the filter should also have the same `aria-activedescendant` attribute and it should also somehow be passed to it's custom render function
- Selected options in the list are not labelled as such. It would be nice to add `aria-selected` or some other attribute to them
